net/usrsock/usrsock_recvfrom.c

```diff
@@ -397,9 +399,13 @@ ssize_t usrsock_recvfrom(FAR struct socket *psock, FAR void *buf, size_t len,
 
       usrsock_setup_datain(conn, inbufs, ARRAY_SIZE(inbufs));
 
+      /* MSG_DONTWAIT is only use in usrsock. */
+
+      flags &= ~MSG_DONTWAIT;
```

Comment:
Should we need handle DONTWAIT like NONBLOCK flag?

Reply:
Yes, we need it.